controller_manager/doc/userdoc.rst

```diff
@@ -83,6 +79,13 @@ update_rate (mandatory; integer)
   Name of a plugin exported using ``pluginlib`` for a controller.
   This is a class from which controller's instance with name "``controller_name``" is created.
 
+Subscribers
+-----------
+
+robot_description (std_msgs/msg/String)
+  String with the URDF string as robot description.
+  This is usually published by the ``robot_state_publisher`` node.
+
```

Comment:
I'm not sure whether this is the way we want to suggest? Intuitively, I would use namespaces for handling this:

```python
control_node = Node(                                                                                                                                                                                                                                                               
    package="controller_manager",                                                                                                                                                                                                                                                  
    executable="ros2_control_node",                                                                                                                                                                                                                                                
    parameters=[robot_controllers],                                                                                                                                                                                                                                                
    output="both",                                                                                                                                                                                                                                                                 
    namespace="rrbot",                                                                                                                                                                                                                                                             
)                                                                                                                                                                                                                                                                                  
robot_state_pub_node = Node(                                                                                                                                                                                                                                                       
    package="robot_state_publisher",                                                                                                                                                                                                                                               
    executable="robot_state_publisher",                                                                                                                                                                                                                                            
    output="both",                                                                                                                                                                                                                                                                 
    parameters=[robot_description],                                                                                                                                                                                                                                                
    namespace="rrbot",                                                                                                                                                                                                                                                             
)
```

And when doing things with remaps, I would skip the initial `/` as this would only be true if the `controller_manager` is `/controller_manager` which is probably not the case in situations where we have multiple descriptions and controller managers:

```python
control_node = Node(                                                                                                                                                                                                                                                               
    package="controller_manager",                                                                                                                                                                                                                                                  
    executable="ros2_control_node",                                                                                                                                                                                                                                                
    parameters=[robot_controllers],                                                                                                                                                                                                                                                
    output="both",                                                                                                                                                                                                                                                                 
    remappings=[('robot_description', '/rrbot/robot_description')]                                                                                                                                                                                                                 
)                                                                                                                                                                                                                                                                                  
robot_state_pub_node = Node(                                                                                                                                                                                                                                                       
    package="robot_state_publisher",                                                                                                                                                                                                                                               
    executable="robot_state_publisher",                                                                                                                                                                                                                                            
    output="both",                                                                                                                                                                                                                                                                 
    parameters=[robot_description],                                                                                                                                                                                                                                                
    namespace="rrbot",                                                                                                                                                                                                                                                             
)
```

However, I could also add both versions to the docs.

Reply:
I agree with you on the initial `/`, I've updated my suggestion.  I thought of the namespace part, but in most cases, the robot_state_publisher node might not exist in the same launch file as that would give a lot of modularity. However, it is good to point this case as well as you suggested

Reply:
I've updated things in [5c89eb3](https://github.com/ros-controls/ros2_control/pull/1410/commits/5c89eb3d04a24a8c6efb97b7cd033988a83022e4).